migrations/20210201160014-fix-opencollective-paymentMethod-currency.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    // Fix Payment Methods currency not matching the Host currency
+    await queryInterface.sequelize.query(`
+      UPDATE "PaymentMethods" as pm
+      SET "currency" = h."currency"
+      FROM "Collectives" as c, "Collectives" as h
+      WHERE c."id" = pm."CollectiveId"
+      AND pm."service" = 'opencollective'
+      AND pm."type" = 'collective'
+      AND pm."deletedAt" IS NULL
+      AND c."deletedAt" IS NULL
+      AND h."id" = c."HostCollectiveId"
+      AND pm."currency" !=  h."currency"
+    `);
+    // Fix some transactions impacted by this
+    // easy cases where t."currency" = t."hostCurrency"
```

Comment:
Intent is to convert transactions exposed by:
https://opencollective-metabase.herokuapp.com/question/190